docs/src/FAQs.md

```diff
@@ -119,3 +119,21 @@ fg #hide
     When setting the `width` and `height` dimensions of each axis manually you
     will need to call `AlgebraOfGraphics.resizetocontent!(fg)` before
     displaying the figure such that each axis is sized correctly.
+
+This `.figure` field can also be used to adjust a plot using `Makie`.
+See the Makie [API reference](http://makie.juliaplots.org/stable/makielayout/reference.html)
+for more information.
+
+```@example combined-3
+using AlgebraOfGraphics, CairoMakie #hide
+
+df = (x = exp.(rand(1000)),) #hide
+hist2 = data(df) * mapping(:x => log => "log(x)") * histogram()  #hide
+fg = draw(hist2; figure = (resolution = (600, 400),))  #hide
+
+lines(fg.figure[1, 1], 0..2pi, cos) #hide
+fg.figure.content[1].xticklabelrotation = Float64(pi/6)
```

Comment:
Thanks for the example! I was thinking that it would be better IMO if here one was modifying the figure directly rather than the axes (the axes should ideally be accessed as `fg.grid[i, j].axis`). Maybe one could change `backgroundcolor` for instance, or any other figure attribute?

Reply:
```suggestion
only(contents(fg.figure[1, 1])).xticklabelrotation = Float64(pi/6)
```
is what the current Makie API prescribes I think.